nss-day-cohort-48/Rare-API-Spicy-Sauce-Scouts
# Login

#### Changes Made
1. Added the log in function for backend. 
​2. Added Register function for backend.

#### Steps to Review
1. Checkout this branch locally.
    git fetch --all
    git checkout login
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Start the server.
4. Test functionality. 
        Unfortunately, this one will need the client side to test properly. Check the other pull request for that information. 
5. View code file.
    > Take a look at the code changes in models/RareUser.py, in views/auth.py, and in rare/urls. Make sure it makes sense. 
    > Confirm no unused code or extraneous comments exist.
